collaboration-and-sharing/README.md

```diff
@@ -133,8 +133,10 @@ The table below summarises the operations that can be performed on media library
 | Download file                    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
 | Download assets as zip           | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
 | View downloads                   | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| View comments                    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| Edit and delete comments         |                    | :heavy_check_mark: | :heavy_check_mark: |
+| View threads and comments        | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Add new comment and replies      |                    | :heavy_check_mark: | :heavy_check_mark: |
+| Edit and delete own comments     |                    | :heavy_check_mark: | :heavy_check_mark: |
+| Modify own reaction on comments  |                    | :heavy_check_mark: | :heavy_check_mark: |
```

Comment:
What about resolving and deleting threads?

Reply:
Added resolve. Delete was already there in the list.